datafusion/physical-plan/src/joins/hash_join.rs

```diff
@@ -1611,9 +1628,9 @@ mod tests {
     async fn join_inner_one_no_shared_column_names() -> Result<()> {
         let task_ctx = Arc::new(TaskContext::default());
         let left = build_table(
-            ("a1", &vec![1, 2, 3]),
+            ("a1", &vec![1, 3, 2]),
```

Comment:
why was this test changed?

Reply:
It's a miss -- the plan was to modify first test for inner join so that the output wouldn't not be equal to sorted output -- I'll better just revert this change and will add separate test with randomly sorted input -- like this one.